src/nvim/message.c

```diff
@@ -137,6 +139,94 @@ int msg_attr(const char *s, const int attr) FUNC_ATTR_NONNULL_ARG(1)
   return msg_attr_keep((char_u *)s, attr, false);
 }
 
+int msg_echo_attr(const char *s, const int attr) FUNC_ATTR_NONNULL_ARG(1)
+{
+  return msg_echo_attr_keep(s, attr, false);
+}
+
+int
+msg_echo_attr_keep(
+    const char *s,
+    const int attr,
+    int keep
```

Comment:
You must not put arguments completely on separate lines unless you absolutely have to. Neither separate type from function name or closing parenthesis from the last argument.